test/src/test/java/org/corfudb/infrastructure/SequencerBenchmark.java

```diff
@@ -0,0 +1,125 @@
+package org.corfudb.infrastructure;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.corfudb.protocols.wireprotocol.Token;
+import org.corfudb.protocols.wireprotocol.TxResolutionInfo;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.test.benchmark.AbstractCorfuBenchmark;
+import org.corfudb.test.benchmark.CorfuBenchmarkState;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.infra.Blackhole;
+
+/**
+ * A set of benchmarks for the sequencer server.
+ *
+ * Created by zlokhandwala on 5/3/19.
+ */
+public class SequencerBenchmark extends AbstractCorfuBenchmark {
+
+    public static class SequencerBenchmarkState extends CorfuBenchmarkState {
```

Comment:
Same for `CorfuBenchmarkState` we don't need it, we already have `@State` annotation provided by JMH